extensions/iam/oauth2/src/test/java/org/eclipse/dataspaceconnector/iam/oauth2/impl/Oauth2ServiceImplTest.java

```diff
@@ -81,7 +81,7 @@ void verifyExpiredToken() {
     @Test
     void verifyValidJwt() {
 
-        var jwt = createJwt("test.audience", new Date(System.currentTimeMillis() - 1000), new Date(System.currentTimeMillis() + 1000));
+        var jwt = createJwt("test.audience", new Date(System.currentTimeMillis() - 1000000), new Date(System.currentTimeMillis() + 1000000));
```

Comment:
Might be to have something like this:
`Date.from(Instant.now().plusSeconds(1000))`

Reply:
looks indeed cleaner